native-src/android/app/src/main/java/com/telerik/localnotifications/Builder.java

```diff
@@ -84,6 +94,17 @@ static Notification build(JSONObject options, Context context, int notificationI
 
     // Notification styles:
 
+    private static void applyNotificationLed(JSONObject options, NotificationCompat.Builder builder) {
+        if (shouldEnableNotificationLed(options)) {
+            try {
+                JSONObject notificationLed = options.getJSONObject("notificationLed");
+                builder.setLights(notificationLed.optInt("ledColor", DEFAULT_NOTIFICATION_COLOR), notificationLed.optInt("ledOn", DEFAULT_NOTIFICATION_LED_ON), notificationLed.optInt("ledOff", DEFAULT_NOTIFICATION_LED_OFF));
+            } catch (JSONException e) {
+                Log.e(TAG, "Error parsing options.notificationLed", e);
```

Comment:
Please feel free to suggest any better way to handle the `JSONException`. :-)

(Currently the only improvement inside my head is to extract the 2 exception handling to a common method)